src/BlazorAdmin/BlazorAdmin.csproj

```diff
@@ -18,7 +18,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\ApplicationCore\ApplicationCore.csproj" />
```

Comment:
Why not reference Core for common things?

Reply:
I thought it is had server classes so all this dll will send to the browser.

Reply:
Ah, good point. It does reduce what we send to the browser. 👍